ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -1111,6 +1414,7 @@ NetworksForm.propTypes = {
   submitCallback: PropTypes.func,
   restrictHeight: PropTypes.bool,
   setActiveOnSubmit: PropTypes.bool,
+  getOnEditCallback: PropTypes.func,
```

Comment:
nit: for naming of props, I suggest the 'on' convention for the name of the prop, I believe just onEditNetwork is a clear enough prop name exposed, to me its already clear its a callback

the intent is that when the component is used, it can read as on edit network, edit network 
<MyComponent onEditNetwork="editNetwork">

or on edit network, handle edit network 
<MyComponent onEditNetwork="handleEditNetwork">

But I believe 'handle' is can be reserved for handling user events like clicks